internal/packetfabric/client.go

```diff
@@ -135,6 +150,65 @@ func (c *PFClient) sendRequest(uri, method string, payload interface{}, resp int
 	return res, nil
 }
 
+func (c *PFClient) sendMultipartRequest(uri, method, fileField, filePath string, payload map[string]string, resp interface{}) (interface{}, error) {
+	var req *http.Request
+	var err error
+	c.Ctx = context.Background()
+	formatedURL := fmt.Sprintf("%s%s", c.HostURL, uri)
```

Comment:
Consider using: https://pkg.go.dev/net/url#JoinPath

Reply:
Adding JoinPath fails for some reason, see https://github.com/PacketFabric/terraform-provider-packetfabric/pull/485/commits/80efad31650c13493e4ed5720ed0ad53fdacf33e, removing for now